partner_firstname/models/res_partner.py

```diff
@@ -192,11 +197,24 @@ def _get_inverse_name(self, name, is_company=False):
 
     @api.multi
     def _inverse_name(self):
-        """Try to revert the effect of :meth:`._compute_name`."""
+        """
+            Try to revert the effect of :meth:`._compute_name`.
+            Generate one write if values are really updated
+        """
+        order = self._get_names_order()
         for record in self:
-            parts = record._get_inverse_name(record.name, record.is_company)
-            record.lastname = parts['lastname']
-            record.firstname = parts['firstname']
+            parts = record._get_inverse_name(record.name, record.is_company, order)
+            values = {}
+            if parts['lastname'] != record.lastname:
+                values.update({
```

Comment:
Why not just: `values["lastname"] = parts["lastname"]`?

And the same below for firstname...

Reply:
Please reply to this remark...